### PR TITLE
some more upgrades

### PR DESCRIPTION
**Pull request by barryoneill**

### CODE REVIEW

File: fs2-aws-kinesis/src/main/scala/fs2/aws/kinesis/Kinesis.scala [hunk 1]
```diff
@@ -155,14 +155,17 @@ object Kinesis {
...
+        new SingleStreamTracker(
+          StreamIdentifier.singleStreamInstance(settings.streamName),
+          settings.initialPositionInStream match {
+            case Left(position) =>
+              InitialPositionInStreamExtended.newInitialPosition(position)
+
+            case Right(date) =>
+              InitialPositionInStreamExtended.newInitialPositionAtTimestamp(date)
+          }
+        )
```
Comment (barryoneill) [inline fs2-aws-kinesis/src/main/scala/fs2/aws/kinesis/Kinesis.scala:168]: unsure if this is correct, but `initialPositionInStreamExtended` is deprecated :/

Comment (barryoneill) [inline fs2-aws-kinesis/src/main/scala/fs2/aws/kinesis/Kinesis.scala:168]: initialPositionInStreamExtended is deprecated

**Comment by semenodm** [comment]
@barryoneill do we need this PR? i'm planning to deprecate this API and suggest using the new builder

**Comment by barryoneill** [comment]
sounds good, I'll close this

**Pull request closed without merging**